x/concentrated-liquidity/lp.go

```diff
@@ -187,6 +192,10 @@ func (k Keeper) updatePosition(ctx sdk.Context, poolId uint64, owner sdk.AccAddr
 		return sdk.Int{}, sdk.Int{}, err
 	}
 
+	if err := k.updateFeeAccumulatorPosition(ctx, poolId, owner, liquidityDelta, lowerTick, upperTick); err != nil {
```

Comment:
this should be initOrUpdateFeeAccumulator or else would error. addressed in https://github.com/osmosis-labs/osmosis/pull/3936

Reply:
Why? there is a separate method to init here that is called in `withdrawPosition`:
https://github.com/osmosis-labs/osmosis/blob/5531b3f9e9a937eb1705d239c8ec18516ee6b4c4/x/concentrated-liquidity/lp.go#L66

Reply:
As discussed in PR #3936, agreed that this is not the case